shared_model/interfaces/commands/impl/add_peer.cpp

```diff
@@ -11,11 +11,17 @@ namespace shared_model {
   namespace interface {
 
     std::string AddPeer::toString() const {
-      return detail::PrettyStringBuilder()
-          .init("AddPeer")
-          .append("peer_address", peer().address())
-          .append("pubkey", peer().pubkey().toString())
-          .finalize();
+      auto builder = detail::PrettyStringBuilder()
+                         .init("AddPeer")
+                         .append("peer_address", peer().address())
+                         .append("pubkey", peer().pubkey().toString());
+
+      auto tls_certificate = peer().tlsCertificate();
+      if (tls_certificate) {
+        builder = builder.append("tls_certificate", *tls_certificate);
```

Comment:
I think we should not include a huge TLS cert into `toString()`. it will bloat our logs. maybe just 
```c++
                         .append("tls_certificate", logger::logBool(tls_certificate));
```

Reply:
oh, let's just make it like
```c++
                         .init("AddPeer")
                         .append(peer().toString())
                         .finalize()
```